jettons/WB.yaml

```diff
@@ -0,0 +1,8 @@
+name: Bakalchuk
+description: $WB is a meme coin on the TON Blockchain.
+image: "https://pumpers.tg/api/startup/tokens/images/mnkerhx2nt4b5ycqgfhgcid9"
+address: "0:ef39c29f72785e150fb40ce3df085afcc3664f307108f4ca0ee0092140b9b335"
+symbol: WB
+social:
+  - "https://t.me/CryptoBakalchuk"
+  - "https://x.com/CryptoBakalchuk"
```

Comment:
![Screenshot_2024-09-21-12-46-51-500_com ton_keeper](https://github.com/user-attachments/assets/a8241613-004e-41ac-b4c2-f90b372d93be)
![Screenshot_2024-09-21-13-50-08-492_com ton_keeper](https://github.com/user-attachments/assets/cfd66cbb-39ca-4fd5-aee6-9073490c481d)

Reply:
How much I have to send